subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -1391,6 +1286,73 @@ By default, Gradle caches changing modules for 24 hours. To change how long Grad
 
 For more details, take a look at the API documentation for api:org.gradle.api.artifacts.ResolutionStrategy[].
 
+[[sec:dependency_management_overview]]
+=== Dependency Management Best Practices
+
+While Gradle has strong opinions on dependency management, the tool gives you a choice between two options: follow recommended best practices or support any kind of pattern you can think of. This section outlines the Gradle project's recommended best practices for managing dependencies.
+
+No matter what the language, proper dependency management is important for every project. From a complex enterprise application written in Java depending on hundreds of open source libraries to the simplest Clojure application depending on a handful of libraries, approaches to dependency management vary widely and can depend on the target technology, the method of application deployment, and the nature of the project. Projects bundled as reusable libraries may have different requirements than enterprise applications integrated into much larger systems of software and infrastructure. Despite this wide variation of requirements, the Gradle project recommends that all projects follow this set of core rules:
+
+
+[[sub:versioning_the_jar_name]]
+==== Put the Version in the Filename (Version the jar)
+
+The version of a library must be part of the filename. While the version of a jar is usually in the Manifest file, it isn't readily apparent when you are inspecting a project. If someone asks you to look at a collection of 20 jar files, which would you prefer? A collection of files with names like `commons-beanutils-1.3.jar` or a collection of files with names like `spring.jar`? If dependencies have file names with version numbers you can quickly identify the versions of your dependencies.
+
+If versions are unclear you can introduce subtle bugs which are very hard to find. For example there might be a project which uses Hibernate 2.5. Think about a developer who decides to install version 3.0.5 of Hibernate on her machine to fix a critical security bug but forgets to notify others in the team of this change. She may address the security bug successfully, but she also may have introduced subtle bugs into a codebase that was using a now-deprecated feature from Hibernate. Weeks later there is an exception on the integration machine which can't be reproduced on anyone's machine. Multiple developers then spend days on this issue only finally realising that the error would have been easy to uncover if they knew that Hibernate had been upgraded from 2.5 to 3.0.5.
+
+Versions in jar names increase the expressiveness of your project and make them easier to maintain. This practice also reduces the potential for error.
+
+[[sub:transitive_dependency_management]]
+==== Manage transitive dependencies
```

Comment:
As a reader I don't understand what this means concretely for me and my Gradle build. It would be good to make this crystal clear. For example:

 * Do this thing. It will avoid problem X and make Y easier
 * You can learn more by reading Z

Also, a build scan image here would be a great way to "show, don't tell".

Reply:
Will be addressed in a different PR.